app/components/Views/AccountConnect/AccountConnect.test.tsx

```diff
@@ -280,7 +283,16 @@ mockGetConnection.mockReturnValue(undefined);
 mockIsUUID.mockReturnValue(false);
 
 describe('AccountConnect', () => {
+  beforeEach(() => jest.clearAllMocks());
   it('renders correctly with base request', () => {
```

Comment:
```suggestion
  it('renders correctly with base request when there is no existing CAIP endowment', () => {
```
? or something like that?

Do we need a scenario testing what it looks like when permissions are merged from a CAIP-25 endowment? No is an okay answer since this component is fairly complicated and covered well by e2e tests